### ISSUE TITLE
Deployment

### ISSUE THREAD

**Issue by OpsRJ** [AdamBien/afterburner.fx#35]
I´ve finished my app and everything is running as it should via eclipse. I´m using an ant script to build the app, but whenever i try to run it, i got: exception in application start method. And exception is cused by: Location is not set. (Probably an FXML load error)

Is there a way to log the the entire exception so i can find out what is causing it and fix it? Or maybe there is a step that i´m not doing to deploy it???

**Comment by AdamBien** [comment]
Just look [http://www.adam-bien.com/roller/abien/entry/igniter_fx_the_afterburner_fx](http://www.adam-bien.com/roller/abien/entry/igniter_fx_the_afterburner_fx). Maven is setup correctly in the project. I test afterburner with maven only so far.

**Comment by achieva** [comment]
Hi Adam,
I encountered the same error.
Running under eclipse is fine; however, when running as an executable jar it failed with the same error message.
During debugging I see that the the URL Resource is NULL when running as an executable jar but not unter eclipse and this caused the error.
 --> this.resource = clazz.getResource(conventionalName);
the resource was NULL.
It is solved when I changed the filename of the fxml file to lower case.

I am building with gradle and spring boot.
Regards

Roger
